hack/generate-crd.sh

```diff
@@ -1,23 +1,15 @@
-#!/usr/bin/bash
+#!/bin/bash
 
 source .bingo/variables.env
 
 set -euo pipefail
 
-MANIFESTS_DIR=${1:-"manifests/${OCP_VERSION}"}
-
 echo "--------------------------------------------------------------"
 echo "Generate k8s golang code"
 echo "--------------------------------------------------------------"
-$OPERATOR_SDK generate k8s
+$CONTROLLER_GEN object paths="./apis/..."
 
 echo "--------------------------------------------------------------"
 echo "Generate CRDs for apiVersion v1"
 echo "--------------------------------------------------------------"
-$OPERATOR_SDK generate crds --crd-version v1
-mv deploy/crds/*.yaml "${MANIFESTS_DIR}"
-
-echo "---------------------------------------------------------------"
-echo "Cleanup operator-sdk generation folder"
-echo "---------------------------------------------------------------"
-rm -rf deploy
+$CONTROLLER_GEN crd:crdVersions=v1 rbac:roleName=elasticsearch-operator paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
I suggest to move these commands into the Makefile under `generate` and eliminate this script entierly. It became really easy to generate objects and crds as the procedures don't need any further processing with bash commands.